pw_scale.m

```diff
@@ -1,11 +1,16 @@
-function [Q, R] = pw_scale( D )
+function [Q, R] = pw_scale( D, options )
 % Scaling method for pairwise comparisons, also for non-balanced
 % (incomplete) designs.
 %
-% [Q, R] = pw_scale( D )
+% [Q, R] = pw_scale( D, use_prior )
 %
 % D - NxN matrix with positive integers. D(i,j) = k means that the
 %     condition i was better than j in k number of trials.
+% options - a cell array with the options. Currently recognized options:
+%	   'prior' - type of the distance prior in the available options are:
+%                'none': do not use prior, 'bounded': , 'gaussian': the
+%                normalised sum of probabilities of observing a difference 
+%                for all compared pairs of conditions. Set to 'none' by default. 
 % Q - the JOD value for each method. The difference of 1 corresponds to
 %     75% of answers selecting one condition over another.
 % R - matrix of residuals. The residuals are due to projecting the
```

Comment:
something is missing in the sentence 

"projecting to one-dimensional quality scale" - my guess